qsr_lib/scripts/example_ros_client.py

```diff
@@ -34,7 +34,8 @@
                "qtcbc": "qtc_bc_simplified",
                "rcc3a": "rcc3_rectangle_bounding_boxes_2d",
                "arg_distance": "arg_relations_distance",
-                "mos": "moving_or_stationary"}
+                "mos": "moving_or_stationary",
+                "multiple": ("cone_direction_bounding_boxes_centroid_2d", "rcc3_rectangle_bounding_boxes_2d", "moving_or_stationary", "qtc_b_simplified")}
```

Comment:
Tried it, the problem with that is, that not all QSRs provide default values for the dynamic_args you can set, e.g.:
```
responded with an error: error processing request: qsr_relations_and_values is uninitialized,
use dynamic_args={'qsr_relations_and_values': <your dictionary of relations and values>in the QSRlib_Request_Message
```

Therefore I would need to define all of those as well. Can do that but I'm not sure it's worth it just to give an example.